source/includes/options-mongosqld.yaml

```diff
@@ -671,4 +671,37 @@ args: <service-name>
 description: |
   Description of the system service which runs :binary:`~bin.mongosqld`.
 optional: true
+---
+program: mongosqld
+name: gssapiHostname
+directive: option
+args: <hostname>
+description: |
+  .. versionadded:: 2.5
+
+  Hostname that serves Kerberos.
```

Comment:
not necessarily the hostname that serves it (there could be many of those). this option determines the hostname part of the kerberos SPN. we can use mongod's security.sasl.hostName as a template for this option